Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-grid/SubmissionsGrid.module.scss

```diff
@@ -37,6 +37,11 @@
     background-color: colors.$light-blue-color;
     color: colors.$white-color;
   }
+
+  .unActiveElement {
```

Comment:
With the previous code, we have been using the <button> element. I replaced it with our custom component Button in order to follow conventions. This change is made due to the proper styling of it.

Reply:
Then move these styles to the base styling of the elements and give it a name, instead of creating a separate style for when the element is not active.

Reply:
transferred to base styling